stubs/pyinstaller/PyInstaller/utils/win32/versioninfo.pyi

```diff
@@ -0,0 +1,51 @@
+from _typeshed import SliceableBuffer
+from collections.abc import Sequence
+from typing import Protocol
+from typing_extensions import Literal, TypeAlias
+
+_FourIntSequence: TypeAlias = Sequence[int] | tuple[int, int, int, int]
```

Comment:
Not sure how useful this is as a `tuple[int, int, int, int]` is a subtype of `Sequence[int]`. What type safety does this give us?